code/game/objects/items/devices/aicard.dm

```diff
@@ -100,6 +100,10 @@
 			. = TRUE
 		if("wireless")
 			AI.control_disabled = !AI.control_disabled
+			if(!AI.control_disabled)
+				AI.interaction_range = null
+			else
+				AI.interaction_range = 0
```

Comment:
Aren't these functionally identical?

Reply:
> Aren't these functionally identical?

nope, 0 means it has a range check of x.y tile +- 0 tiles. aka you can only interact with things your directly standing on. whereas null allows you to bypass the range check completely granting you unlimited range. a carded AI without wireless on should not have any range, so its set to 0